modules/creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,195 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-with-aws-kms-key"]
+= Creating a ROSA cluster using custom AWS KMS key
+
+You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that will be used to encrypt both the root volumes of nodes as well as persistent volumes for mounted EBS claims.
```

Comment:
```suggestion
You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that is used to encrypt node root volumes and persistent volumes for mounted EBS claims.
```

Reply:
Do we need to spell out KMS?

Reply:
Already spelt out in the assembly and module content unlikely to be reused.